### ISSUE TITLE
prune redundant branches

### ISSUE THREAD

**Issue by digiltd** [squeezebox-googlemusic/squeezebox-googlemusic#8]
Thanks for your work to bring this plugin up to date.

I have installed it over my old version and it seems to have worked without any issues.

Would it be an idea to do some housekeeping and remove some of the many branches? 

Perhaps with a final "just in case" fork/export :)

**Comment by askvictor** [comment]
Hadn't even noticed them; I'll have a look at some point to see if there's anything worth salvaging, but not imminently (they're not doing any harm I guess :)

**Comment by digiltd** [comment]
Cool, thanks :)

I spotted them as I was being nosey and having a look for a dev branch to see what potential tweaks you were working on. If it helps I have had a look and the v0.3 and v0.4 branches have been merged already into master, with the feature branches merged into the "v" branches.

Ideally it would be good to just have a master and dev branch, with new feature branches created as required (and deleted when merged back into dev). But as you say, there is no real harm in keeping them... for a little while :)
